models/_sector/dex/trades/base/platforms/sushiswap_v2_base_base_trades.sql

```diff
@@ -0,0 +1,21 @@
+{{
+    config(
+        schema = 'sushiswap_v2_base',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    uniswap_compatible_v3_trades(
+        blockchain = 'base',
+        project = 'sushiswap',
+        version = '2',
```

Comment:
The version is ambiguous for `sushiswap_v2_base.base_trades`,
as the source using here is `sushiswap_v3_base.UniswapV3Pool_evt_Swap`, and `sushiswap_base.trades` uses version `1`.

If we want to keep consistent with `dex.trades`, we can use version `1` here, otherwise we may change the corresponding seed's version.

Reply:
Existing `sushiswap_base_trades` is incorrect imo - it should show as version **2**, but I agree that source naming is a mess. In general: sushi v1 is based on uni v2 and sushi v2 on uni v3, so in the case of base chain we have the following sources:

sushi v1 - not added to models yet:
`sushiswap_base.Pair_evt_Swap`
`sushiswap_base.Factory_evt_PairCreated` (contract decoding: Pending as I accidently set it as created by a factory contract..)

sushi v2 - added here, but with confusing naming as you noticed:
`sushiswap_v3_base.UniswapV3Pool_evt_Swap`
`sushiswap_base.UniswapV3Factory_evt_PoolCreated`

Reply:
Noted, then we may modify the corresponding base_trades seed to version `'2'`.